frontend/www/js/omegaup/components/user/Profilev2.vue

```diff
@@ -0,0 +1,268 @@
+                <omegaup-grid-paginator
+                  v-bind:columns="1"
+                  v-bind:items="contests"
+                  v-bind:items-per-page="15"
+                >
+                  <template slot="table-header">
+                    <thead>
+                      <tr>
+                        <th>{{ T.profileContestsTableContest }}</th>
+                        <th class="numericColumn">
+                          {{ T.profileContestsTablePlace }}
+                        </th>
+                      </tr>
+                    </thead>
+                  </template>
+                </omegaup-grid-paginator>
+              </div>
+              <div
+                class="tab-pane fade show active"
+                role="tab"
+                aria-labelledby="nav-user-info-tab"
+                v-if="selectedTab == 'data'"
+              >
+                <omegaup-user-basicinfo
+                  v-bind:profile="profile"
+                  v-bind:rank="rank"
+                ></omegaup-user-basicinfo>
+              </div>
+              <div
+                class="tab-pane fade show active"
+                role="tab"
+                aria-labelledby="nav-charts-tab"
+                v-if="selectedTab == 'charts'"
+              >
+                <omegaup-user-charts
+                  v-bind:data="charts"
+                  v-bind:username="profile.username"
+                  v-bind:periodStatisticOptions="this.periodStatisticOptions"
+                  v-on:emit-update-period-statistics="
+                    (profileComponent, categories, data) =>
+                      $emit(
+                        'update-period-statistics',
+                        profileComponent,
+                        categories,
+                        data,
+                      )
+                  "
+                  v-bind:aggregateStatisticOptions="
+                    this.aggregateStatisticOptions
+                  "
+                  v-on:emit-update-aggregate-statistics="
+                    profileComponent =>
+                      $emit('update-aggregate-statistics', profileComponent)
+                  "
+                  v-if="charts"
+                ></omegaup-user-charts>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+</template>
+
+<style>
+a:hover {
+  cursor: pointer;
+}
+.card-header {
+  font-size: 1rem !important;
+}
+th.numericColumn {
+  text-align: right;
+}
+* {
+  font-size: 1rem;
+}
+</style>
+
+<script lang="ts">
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { omegaup } from '../../omegaup';
+import T from '../../lang';
+import { Chart } from 'highcharts-vue';
+import user_BasicInfo from './BasicInfov2.vue';
+import user_Username from './Username.vue';
+import user_Charts from './Chartsv2.vue';
+import badge_List from '../badge/List.vue';
+import gridPaginator from '../common/GridPaginator.vue';
```

Comment:
```suggestion
import common_GridPaginator from '../common/GridPaginator.vue';
```

la convención es usar `directorio_NombreDelComponente`.